tests/bgp/test_bgp_speaker.py

```diff
@@ -280,7 +280,7 @@ def bgp_speaker_announce_routes_common(common_setup_teardown, tbinfo, duthost,
     announce_route(ptfip, lo_addr, peer_range, vlan_ips[0].ip, port_num[2])
 
     logger.info("Wait some time to make sure routes announced to dynamic bgp neighbors")
-    assert wait_until(90, 10, 0, is_all_neighbors_learned, duthost, speaker_ips), \
+    assert wait_until(120, 10, 0, is_all_neighbors_learned, duthost, speaker_ips), \
```

Comment:
Could you please share the typical and maximum value we have seen?  thanks

Reply:
Hi @lipxu , typically it is within 90s but sometimes we have seen it in the range of 100s-115s in our 202311 runs

Reply:
thanks for your information @AnantKishorSharma , do you know if we have the timeout configuration or something like that? If so, we could set the waiting value to match the timeout and add a comment indicating that if assert here next time, it should be a real issue.

Reply:
afaik there is no timeout configuration that can be used for this purpose here..